plugins/woocommerce/includes/abstracts/abstract-wc-session.php

```diff
@@ -72,17 +74,17 @@ public function __set( $key, $value ) {
 	/**
 	 * Magic isset method.
 	 *
-	 * @param mixed $key Key to check.
+	 * @param string $key Key to check.
 	 * @return bool
 	 */
 	public function __isset( $key ) {
-		return isset( $this->_data[ sanitize_title( $key ) ] );
+		return isset( $this->_data[ sanitize_key( $key ) ] );
```

Comment:
All other data access functions use `sanitize_key` so this looks like a mistake.

Reply:
Sanitize key calls `strtolower`, which wasn't the case for `sanitize_title`. So there might be some side effects with this change. Should we communicate this as a breaking change?

Reply:
Isn't the key internal? See here there are other uses of `sanitize_key` in the magic methods https://github.com/woocommerce/woocommerce/blob/157ea29fca12d91fb2b048e2fe300e8baabec910/plugins/woocommerce/includes/abstracts/abstract-wc-session.php#L74-L132 however I think those are quite recently added by another team. See https://github.com/woocommerce/woocommerce/commit/b6ed620bb64e580645190a930c56682406b88c75

Reply:
can extensions set other keys? if they do, then `__isset ` might be returning a different value, because it is checking on a different key than before. 

I’m not sure if this will have a real-world impact, I only wanted to ask if you think we should put it in the changelog.

Reply:
`__isset` seems to only check within `$this->_data`, just like __unset, get and set. Given the other 3 methods use `sanitize_key` I don't see a problem. 

@prettyboymp Is there a reason you changed the others to use `sanitize_key` but not this method?